internal/backend/runtime/omni/controllers/omni/internal/task/machine/poll.go

```diff
@@ -58,6 +60,12 @@ var resourcePollers = map[string]machinePollFunction{
 var machinePollers = map[string]machinePollFunction{
 	"version": pollVersion,
 	"disks":   pollDisks,
+
+	// we do not use a resource poller for the secure boot status, as we want to mark
+	// secure boot explicitly to disabled (contrary to leaving it nil) if the feature is not available (i.e., older Talos versions).
+	//
+	// resourcePollers skip polling the resource if it is not defined on the Talos API.
+	"secureBootStatus": pollSecureBootStatus,
```

Comment:
I guess it should also do it only once for the machine :thinking: 
As soon as we read it, no need to continue polling as it can't be changed.

Reply:
as discussed, it will only poll when the collect task crashes, but not periodically. so it is good enough for now.